server/app/filters/DisableCachingFilter.java

```diff
@@ -11,8 +12,14 @@
 public class DisableCachingFilter extends EssentialFilter {
   private final Executor exec;
 
+  private final ImmutableList<String> staticAssetPaths =
+      ImmutableList.of("/public/", "/assets/", "favicon.ico", "/favicon.ico");
+
+  @Inject private play.Environment environment;
```

Comment:
suggestion (blocking): we use constructor injection everywhere else in the app to call `checkNotNull`, please use that pattern here as well

Reply:
Done. Yeah, sorry.  This was from play's example.